backend/src/kitconcept/intranet/services/votes.py

```diff
@@ -0,0 +1,35 @@
+from plone import api
+from plone.protect.interfaces import IDisableCSRFProtection
+from plone.restapi.services import Service
+from zope.interface import alsoProvides
+
+
+class VotesPost(Service):
+    """Save userID inside votes field.
+    This is used to show object likes.
+    If user clicks again, it removes the like/vote."""
+
+    def reply(self):
+        username = api.user.get_current().id
+
+        # Anonymous user can't add a like (can't vote)
+        if username == "acl_users":
+            return "No logged-in user."
```

Comment:
```suggestion
        if api.user.is_anonymous():
            raise Unauthorized("Must be logged in to vote.")

        username = api.user.get_current().getId()
```

(and you'll need to import Unauthorized from zExceptions at the top)